### DISCUSSION TITLE
feat[vscode]: `eslint.rules.customizations` equivalent

### DISCUSSION THREAD

**Discussion by juliusmarminge** [biomejs/biome#4412] [General]
Support the equivalent of [eslint.rules.customizations](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint#settings-options).

I like having the distinction of type-errors being red and linting stuff be yellow

![CleanShot 2024-10-29 at 00 04 02@2x](https://github.com/user-attachments/assets/ca9b1a61-16bf-488c-84d1-e3afc01907fc)

**Comment by Conaclos** [comment]
In Biome 2.0, we are planning to change the default severity level to `warn` for all rules, except `correctness` rules. I don't think we need a such option.

**Reply by ematipico** [reply]
I believe the proposal is different. OP wants to change the severity of a rule ONLY in the LSP (editor), but keep the original severity in the CLI

**Reply by juliusmarminge** [reply]
yea i want my linter to error but keep them separate from type errors in the editor.

I only tend to either have a lint rule on (by erroring) or off so the distinction of lint warnings and lint errors isn't useful to me. That's why I like the eslint vscode option to turn the severity down so they're all yellow
